packages/teleport/src/services/user/user.test.ts

```diff
@@ -67,7 +67,8 @@ test('fetch user context, null response gives proper default values', async () =
   });
 
   expect(response.accessStrategy).toStrictEqual(defaultStrategy);
-  expect(response.requestableRoles).toStrictEqual([]);
+  expect(response.accessCapabilities.requestableRoles).toStrictEqual([]);
+  expect(response.accessCapabilities.suggestedReviewers).toStrictEqual([]);
```

Comment:
What about this -> `expect(response).toStringEqual(needed)` ?

Reply:
i revised this test to test for matching response with expected object